### PR TITLE
Ft docs enhancement

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- Removes the coverpage of the docs
- Adds the respective links for the docs on the new landing page, thus
- Making the docs site more intuitive to use

#### Status of maturity (all need to be checked before merging):
- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] I've updated corresponding documentation for the changes in this PR.
- [x] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?
- Checkout the landing page in the [link](https://ft-docsenhancement-docs-preview-w7kzhvlewq-ew.a.run.app/) on PR preview comment

#### Screenshots (optional)
image

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
         name: "Home",
         repo: "https://github.com/airqo-platform/AirQo-frontend/tree/master/docs",
         themeColor: "#145dff",
-        coverpage: true,
+        coverpage: false,
         loadNavbar: true,
```
Comment (123MwanjeMike) [inline docs/index.html:28]: Turning off the coverpage

File: docs/cypress/e2e/tests/index.cy.js [hunk 2]
```diff
@@ -1,54 +1,54 @@
...
       .contains("Clean Air for all African Cities");
   });
 
-  it("Verify GitHub Call-To-Action is functional", () => {
+  it.skip("Verify GitHub Call-To-Action is functional", () => {
     cy.get('[href="https://github.com/airqo-platform"]').click();
   });
 
-  it("Verify Explore Data Call-To-Action is functional", () => {
+  it.skip("Verify Explore Data Call-To-Action is functional", () => {
```
Comment (123MwanjeMike) [inline docs/cypress/e2e/tests/index.cy.js:23]: Skip coverpage tests

File: docs/cypress/e2e/tests/index.cy.js [hunk 3]
```diff
@@ -1,54 +1,54 @@
...
+  });
+
   it("Verify Side Bar menu sections exist", () => {
     cy.get(".sidebar > .search").should("be.exist");
     cy.get(".sidebar > .app-name").should("be.exist");
     cy.get(".sidebar > .sidebar-nav").should("be.exist");
   });
 
-  it("Verify Side Bar Navigation menu Links are present", () => {
+  it("Verify Navigation Links are present", () => {
```
Comment (123MwanjeMike) [inline docs/cypress/e2e/tests/index.cy.js:37]: Update test description to match newly added anchor tags

File: docs/README.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
 
 With this reference, you can explore and interact with all publicly availed AirQo resources.
+
+Jump straight to your desired documentation
+
+- [API](api/)
+- [Calibration](calibration/)
+- [Hardware](hardware/)
+- [Mobile App](mobile_app/)
+- [Platform](platform/)
```
Comment (123MwanjeMike) [inline docs/README.md:11]: Add all the respective guides in the documentation

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @123MwanjeMike

**Pull request merged**